indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1439,112 +1440,65 @@ def firstPage(self, c, doc):
     def getBody(self, story=None, indexedFlowable={}, level=1 ):
         if not story:
             story = self._story
-
         style = ParagraphStyle({})
         style.fontSize = 12
-        text = i18nformat(""" _("Registrant ID") : %s""")%self._reg.getId()
-        p = Paragraph(text, style, part=escape(self._reg.getFullName()))
-        story.append(p)
 
-        story.append(Spacer(inch, 0.5*cm, part=escape(self._reg.getFullName())))
+        registration = self._reg
+        data = registration.data_by_field
+        full_name = "{first_name} {last_name}".format(first_name=registration.first_name.encode('utf-8'),
+                                                      last_name=registration.last_name.encode('utf-8'))
+
+        def _append_text_to_story(text, space=0.2, indexed_flowable=False):
+            p = Paragraph(text, style, full_name)
+            if indexed_flowable:
+                indexedFlowable[p] = {"text": full_name, "level": 1}
```

Comment:
WTF. Modifying a mutable default argument. :volcano: (I know, it's legacy :hankey: )